Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Run 'make docker' to build a statically linked vis executable!
-FROM i386/alpine:3.13
```

Comment:
Wouldn’t it be better to have here `alpine:latest`?

Reply:
Not sure... I think the idea here was to have reproducibility. However packages are updated within one branch as well.

Also we had breaking changes with alpine updates, see 0ba252f08e9ef7b62c442eb91b36822d654f5b04 for example. With a commit for every version an issue shows up in bisecting.

Reply:
:+1: then, but it should be the Biblical 3.16 now, shouldn't it?

Reply:
Last commit is for 3.16.
Let's keep everything between, again for bisecting.